src/schema/AdxSchemaResolver.ts

```diff
@@ -80,14 +80,17 @@ export class AdxSchemaResolver {
       );
 
       return schema.OrderedColumns.reduce((columns: AdxColumnSchema[], column) => {
-        const schemaForDynamicColumn = schemaByColumn[column.Name];
+        if (column.CslType !== 'dynamic') {
```

Comment:
one of our goals for this plugin is to increase the code coverage. Can you add a new test file to test (at least) this new functionality?

Reply:
I will work on getting some tests in for this (and potentially expanding code coverage in a future PR).